validator/gulpjs/README.md

```diff
@@ -0,0 +1,31 @@
+## Installation
+
+Install package with NPM and add it to your development dependencies:
+
+```
+npm install --save-dev gulp-amphtml-validator
+```
+
+## Usage
+
+```js
+const ampValidator = require('gulp-amphtml-validator');
+
+gulp.task('amphtml:validate', () => {
+  return gulp.src(['samples/*.html'])
```

Comment:
As it is in the readme and only for documentation purposes simply using `*.html` is probably best.